jitsi-utils/pom.xml

```diff
@@ -51,66 +43,52 @@
     <dependency>
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
-      <version>28.2-jre</version>
+      <version>29.0-jre</version>
     </dependency>
     <dependency>
       <groupId>org.reflections</groupId>
       <artifactId>reflections</artifactId>
       <version>0.9.11</version>
+      <exclusions>
+        <exclusion>
+          <groupId>com.google.guava</groupId>
+          <artifactId>guava</artifactId>
+        </exclusion>
+      </exclusions>
     </dependency>
-    <!-- https://mvnrepository.com/artifact/com.github.spotbugs/spotbugs-annotations -->
     <dependency>
       <groupId>com.github.spotbugs</groupId>
       <artifactId>spotbugs-annotations</artifactId>
-      <version>${spotbugs.version}</version>
       <optional>true</optional>
     </dependency>
 
     <dependency>
       <groupId>junit</groupId>
       <artifactId>junit</artifactId>
-      <version>4.11</version>
+      <version>4.13</version>
       <scope>test</scope>
     </dependency>
   </dependencies>
-
+  
   <build>
     <plugins>
-      <plugin>
```

Comment:
Did you mean to drop the ktlint?

Reply:
Not sure I follow, this comment is on the java package and ktlint is in jitsi-utils-kotlin:
https://github.com/jitsi/jitsi-utils/pull/60/files/4a6d13e7698c49458ac89d5f5e12ae374bbc9d3a..83bb7396074e495693957870f89d0b3fb8fd44aa#diff-a11cc5b682899cc8f742fc654588e716R78

Reply:
Oh, I see, you changed from 'antrun ktlint' to 'ktlint-maven-plugin'.  Is the behavior the same?

I don't know why GitHub reported this comment as being on `jitsi-utils/pom.xml`, I commented on `jitsi-utils-kotlin/pom.xml`

Reply:
I didn't change any Kotlin code, but I ran the plugin to format code and it didn't change, i.e. reformat, anything. The maven plugin uses the same libraries that were invoked from antrun.